.gitignore

```diff
@@ -0,0 +1,145 @@
+# Keep editor-specific, non-project specific ignore rules in global .gitignore:
```

Comment:
Is all of this necessary to include this Template in the repo?

Reply:
i just copied the one from the influxdb repo.